src/components/project/home-block.vue

```diff
@@ -13,11 +13,13 @@ except according to the terms contained in the LICENSE file.
   <div class="project-home-block">
     <div class="title">
       <router-link :to="projectPath(project.id)">{{ project.name }}</router-link>
-      <span v-if="project.keyId" class="encrypted badge"
-        :title="$t('encryptionTip')">
-        <span class="icon-lock"></span>
-        {{ $t('encrypted') }}
+      <template v-if="project.keyId">
+        <span class="encrypted badge" aria-hidden="true" v-tooltip.sr-only>
+          <span class="icon-lock"></span>
+          {{ $t('encrypted') }}
         </span>
+        <span class="sr-only">{{ $t('encryptionTip') }}</span>
```

Comment:
Similar to my comment above in `FormRow`, since screen readers will read `$t('encryptionTip')`, I don't think it's useful for them to also read `$t('encrypted')` immediately beforehand — hence `aria-hidden="true"` on the first `<span>` and no `&nbsp;` in the `.sr-only` element.